app/MasterTemplate.hs

```diff
@@ -39,7 +39,7 @@ header :: T.Text -> H.Html
 header page =
     H.nav ! A.class_ "row header" $ do
         H.img ! A.id "courseography-header" ! A.src "/static/res/img/logo.png"
-             ! H.customAttribute "context" (textValue page)
+            ! H.customAttribute "context" (textValue page)
```

Comment:
revert this change (in fact, it's no longer an issue on Master because of your other recent PR!)